src/pages/meetingsPage.jsx

```diff
@@ -1,132 +1,191 @@
+            {pillar}
+          </h2>
+          <h1 className="font-karla text-[24px] text-center font-bold leading-[28.06px] tracking-[0.12em] ml-20" style={{color: '#7D7D7D'}}>
+                  {date}
+          </h1>
+        </div>
+
+        <p className="font-inter text-base leading-7 text-left">
           {description}
         </p>
       </div>
     </div>
   );
-};
+}
 
 export function MeetingsPage() {
-  const [activeTab, setActiveTab] = useState('FALL');
+  const [activeQuarter, setQuarter] = useState("Fall")
 
-  const sections = [
-    {
-      title: "Insert Title Here",
-      date: "XX/XX/XXXX",
-      pillar: "PILLAR",
-      description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
-    },
-    {
-      title: "Insert Title Here",
-      date: "XX/XX/XXXX",
-      pillar: "PILLAR",
-      description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
-    },
-    {
-      title: "Insert Title Here",
-      date: "XX/XX/XXXX",
-      pillar: "PILLAR",
-      description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
-    },
-    {
-      title: "Insert Title Here",
-      date: "XX/XX/XXXX",
-      pillar: "PILLAR",
-      description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."
+  function QuarterButton({Quarter}){
```

Comment:
Great job with this mechanic i think its in this section but its a small thing with the length of the fall winter and spring buttons in how they aren't the same size